commons/src/main/java/com/expedia/www/haystack/pipes/commons/kafka/config/ProjectConfiguration.java

```diff
@@ -0,0 +1,101 @@
+package com.expedia.www.haystack.pipes.commons.kafka.config;
+
+import com.expedia.www.haystack.commons.config.ConfigurationLoader;
+import com.typesafe.config.Config;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ProjectConfiguration {
+
+    private final Config haystackConfig;
+    private static ProjectConfiguration projectConfiguration = null;
+    private static KafkaConsumerConfig kafkaConsumerConfig = null;
+    private static KafkaProducerConfig kafkaProducerConfig = null;
+    private static FirehoseConfig firehoseConfig = null;
+    private static PipesConfig pipesConfig = null;
+    private static HttpPostConfig httpPostConfig = null;
+
+    private ProjectConfiguration() {
```

Comment:
Why configuration logic and the entities are in commons module? Those should be with each individual modules